examples/geometry/surfaces/exampleEstimatorFromSurfelFunctors.cpp

```diff
@@ -117,7 +117,7 @@ int main(  )
 
```

Comment:
I have an error but not related to this PR: 
[ERR] Assertion Error - assertion (( false && "[ConstAlias::ConstAlias( T&& )] Const-aliasing a rvalue ref has no meaning. Consider Clone instead." )) failed in DGtal::ConstAlias<DGtal::CanonicSCellEmbedder<DGtal::KhalimskySpaceND<3, int> > >::ConstAlias(T &&) [T = DGtal::CanonicSCellEmbedder<DGtal::KhalimskySpaceND<3, int> >]: /Users/kerautre/EnCours/DGtal/src/DGtal/base/ConstAlias.h(259)